src/ansys/fluent/core/workflow.py

```diff
@@ -979,21 +979,26 @@ def _add_child(self, state: Optional[dict] = None) -> None:
         state = state or {}
         if self._dynamic_interface:
             state.update({"add_child": "yes"})
-            self.arguments.set_state(state)
+            self.arguments.update_dict(state)
         else:
             state.update({"AddChild": "yes"})
-            self._task.Arguments.set_state(state)
+            self._task.Arguments.update_dict(state)
 
-    def add_child_and_update(self, state=None):
+    def add_child_and_update(self, state=None, defer_update=None):
         """Add a child to this CompoundTask and update.
 
         Parameters
         ----------
         state : Optional[dict]
             Optional state.
+        defer_update: Optional[bool]
+            Flag to defer update.
         """
         self._add_child(state)
-        self._task.AddChildAndUpdate()
+        if defer_update is None:
+            self._task.AddChildAndUpdate()
+        else:
+            return self._task.AddChildAndUpdate(DeferUpdate=defer_update)
```

Comment:
Are the behaviours `AddChildAndUpdate()` and `AddChildAndUpdate(DeferUpdate=False)` different? If not, the default value of `defer_update` can be set to False.